src/app/components/work-item-list-entry/work-item-list-entry.component.html

```diff
@@ -8,7 +8,7 @@
 	<!-- info area -->
   <div class="list-view-pf-main-info">
     <div class="list-view-pf-left type f8-wi__list-witype">
-      <span almIcon [iconType]="workItem.attributes['system.state']" class="color-grey pull-left"></span>
+      <span almIcon [iconType]="workItem.attributes['system.state']" class="pull-left" title="{{workItem.attributes['system.state']}}"></span>
```

Comment:
Was removing the class ```color-grey``` intentional?

Reply:
Yes. Because class 'color-grey' is not needed here as colors are being fetched from ngx-widgets.

Reply:
Cool, I am good with it.